examples/platform/infineon/cyw30739/TestEventTriggerHandler.h

```diff
@@ -0,0 +1,42 @@
+/*
+ *
+ *    Copyright (c) 2024 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <app/TestEventTriggerDelegate.h>
+
+namespace chip {
+namespace DeviceLayer {
+namespace Infineon {
+namespace CYW30739 {
+
+class TestEventTriggerHandler : public chip::TestEventTriggerHandler
+{
+public:
+    static constexpr uint64_t kFillUpEventLoggingBuffer = 0xffff'ffff'1388'0000;
```

Comment:
I'm definitely not blocking the PR on this, but I almost wonder if this could apply more generally. Other platforms may well find this useful, and the general diagnostics cluster is a REALLY good choice here - it's mandatory, and it looks like you're using a standard API.

Reply:
How about I create a class `GenericEventManagementTestEventTriggerHandler` in `src/app/GenericEventManagementTestEventTriggerHandler.[hc]` for a generic implementation to fill event logging buffers?

Reply:
I moved common trigger logic to the `GenericEventManagementTestEventTriggerHandler` class.

Reply:
Thanks - this is great.